packages/carousel/src/Carousel/Carousel.tsx

```diff
@@ -10,30 +10,34 @@ import {
   SlideProps,
 } from './types';
 
-export const Carousel: CarouselProps = ({ children, showDots = true, showArrows = false, ...props }) => {
+export const Carousel: CarouselProps = ({ children, showDots = true, showArrows = false, visibleSlides, ...props }) => {
   const [totalSlides] = Children.map(children, (child) => {
     if (isValidElement(child) && child.type === Carousel.Slider) {
       return Children.count(child.props.children) as number;
     }
   });
   if (!totalSlides) return;
 
-  const renderDots = (totalSlides: number) => {
+  const renderDots = (totalSlides: number, visibleSlides: number) => {
```

Comment:
Do we need to pass the `visibleSlides` to the `renderDots` or can we just use the value from the properties?

Reply:
either one. Any preference? 

I passed it because `totalSlides` is also available on the outer scope but is still being passed as an argument to the `renderDots` func.